emacs.html.markdown

```diff
@@ -0,0 +1,320 @@
+---
+tool: emacs
```

Comment:
Before the ```tool: emacs``` line, please add the following line

```
category: tool
```

Reply:
Done. Sorry I missed that.